src/basecamp.cpp

```diff
@@ -100,10 +100,16 @@ void basecamp::add_expansion( const std::string &terrain, const tripoint &new_po
 
 void basecamp::define_camp( npc &p )
 {
-    pos = p.global_omt_location();
-    sort_points = p.companion_mission_points;
+    define_camp( p.global_omt_location() );
+}
+
+void basecamp::define_camp( const tripoint omt_pos )
+{
+    pos = omt_pos;
+    //I don't think I need this line, because it has no obvious purpose, but I will leave it in for now
+    //sort_points = p.companion_mission_points;
```

Comment:
This is legacy support code, for savegames from the time when all the faction camp data was saved in the overmap tile or the camp manager.  It should be in the previous version of this function.